From a27682be44d5075cfb4b61ef652d9f4b5462c223 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 12 Nov 2019 10:57:29 +0800
Subject: [PATCH 10/37] MLK-22899 ARM: imx: Add support for Fast Mix off on
 i.MX6SLL

commit 28b2a403048e3319c2c5ce8781bcf2f660e4db83 from
https://source.codeaurora.org/external/imx/linux-imx.git

MMDC offset/value array MUST be assinged to i.MX6SLL power management
SoC data structure to support MMDC restore for suspend with Fast Mix
power off.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Jacky Bai <ping.bai@nxp.com>
(cherry picked from commit 53813106a9565b23c820fa788f49ab4b086f2c2b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/pm-imx6.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-imx/pm-imx6.c b/arch/arm/mach-imx/pm-imx6.c
index d62794f31b83..cbe52d618332 100644
--- a/arch/arm/mach-imx/pm-imx6.c
+++ b/arch/arm/mach-imx/pm-imx6.c
@@ -492,6 +492,8 @@ static const struct imx6_pm_socdata imx6sll_pm_data __initconst = {
 	.pl310_compat = "arm,pl310-cache",
 	.mmdc_io_num = ARRAY_SIZE(imx6sll_mmdc_io_offset),
 	.mmdc_io_offset = imx6sll_mmdc_io_offset,
+	.mmdc_num = ARRAY_SIZE(imx6sll_mmdc_lpddr3_offset),
+	.mmdc_offset = imx6sll_mmdc_lpddr3_offset,
 };
 
 static const struct imx6_pm_socdata imx6sx_pm_data __initconst = {
-- 
2.17.1

